src/Components/EditorTabs/IconEditorTab.js

```diff
@@ -5,7 +5,7 @@ import IconComponent from "../Icon";
 import IconSearch from "./IconEditorTab/IconSearch";
 import IconSearchResults from "./IconEditorTab/IconSearchResults";
 
-function IconEditorTab({ widget }) {
+const IconEditorTab = Scrivito.connect(({ widget }) => {
```

Comment:
* [ ] FYI `export default const` would be a more concise notation. So far we're only using the form `export default function` but I don't see why not `const`?

Reply:
If I try to do `export default const IconEditorTab` I get the following eslint error in my editor:

> Parsing error: Only expressions, functions or classes are allowed as the `default` export.

Reply:
Sorry, I somehow confused `export const` (valid) with `export default const` (invalid).